### ISSUE TITLE
docker attach to old running container after live restoring error out with `resize returned error: rpc error: code = 3 desc = invalid argument`

### ISSUE THREAD

**Issue by coolljt0725** [docker/docker#24302]
**Output of `docker version`:**

```
Client:
 Version:      1.12.0-dev
 API version:  1.25
 Go version:   go1.6.2
 Git commit:   7b2c4df
 Built:        Mon Jul  4 07:47:16 2016
 OS/Arch:      linux/amd64
 Experimental: true

Server:
 Version:      1.12.0-dev
 API version:  1.25
 Go version:   go1.6.2
 Git commit:   7b2c4df
 Built:        Mon Jul  4 07:47:16 2016
 OS/Arch:      linux/amd64
 Experimental: true

```

**Output of `docker info`:**

```
(paste your output here)
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Steps to reproduce the issue:**
1. start daemon with `--live-restore` and start a container
2. restart the daemon
3. try to attach to the old running container, the daemon log is:
````

ERRO[0018] Handler for POST /v1.25/containers/946a702de687/resize returned error: rpc error: code = 3 desc = invalid argument
DEBU[0018] Calling POST /v1.25/containers/946a702de687/resize?h=37&w=198
ERRO[0018] Handler for POST /v1.25/containers/946a702de687/resize returned error: rpc error: code = 3 desc = invalid argument

````

try a fix https://github.com/docker/containerd/pull/283

**Comment by coolljt0725** [comment]
@thaJeztah  I think this should be fixed in 1.12, so I added a 1.12.0 milestone.

**Comment by thaJeztah** [comment]
Thanks!

Ping @mlaventure @tonistiigi PTAL
